include/ccf/service/tables/security_policies.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ccf/service/map.h"
+
+using DigestedPolicy = std::array<uint8_t, 32>;
```

Comment:
Should this be a `crypto::Sha256Hash` directly?

Reply:
Yeah assuming it is serialised nicely?

Reply:
It is now serialised nicely